pyccl/tests/test_tkkssc.py

```diff
@@ -103,8 +137,112 @@ def test_tkkssc_errors():
         ccl.halos.halomod_Tk3D_SSC(COSMO, hmc, P1,
                                    prof34_2pt=P2)
 
+    # No normalization for number counts profile
+    with pytest.raises(ValueError):
+        ccl.halos.halomod_Tk3D_SSC(COSMO, hmc, P2, normprof1=False)
+        ccl.halos.halomod_Tk3D_SSC(COSMO, hmc, P1, prof2=P2, normprof2=False)
```

Comment:
You need to add `with pytest.raises` before all of these, otherwise I'm pretty sure only the first one is being called (coveralls complains that you're not testing the other cases)